### PR TITLE
CR / QA Sigs: add links to comments

### LINKED ISSUE #291: Add links to CR and QA Signatures
## Description

I realized we currently don't have links for the `CR` and `QA` stamps like in the old UI.

https://user-images.githubusercontent.com/22064420/162491315-2f91051c-3a82-4075-a22f-51bb1f59f2c3.mp4

**Reference:** https://github.com/iFixit/pulldasher/pull/287#issuecomment-1093029785

### PR DESCRIPTION
**Pull request by danielbeardsley**
Have the checkmarks link to the signature comments they came from.

Closes #291

### CODE REVIEW

**Comment by deltuh-vee** [comment]
QA 🎬 
QA and CR checkboxes now link to their respective Github comments.

deploy_block ❓   
There are some broken links that only take you to the Github issue but not the specific comment:
* Metz's CR on https://github.com/iFixit/valkyrie/pull/650
* Ian's and Metz's CRs on https://github.com/iFixit/ifixit/pull/42405
* Michelle's CR on https://github.com/iFixit/ifixit/pull/42608

**Comment by danielbeardsley** [comment]
Wow, good job noticing @deltuh-vee !  You found a bug! That was actually present in the old UI too. It's because those are *reviews*, not regular comments. When handed to the frontend, they've been stripped of their "review" identity and the frontend assumes they are normal comments.

We should add a `source_type` column in the `pull_signatures` table, but until we do, this was an easy work-around.

un_deploy_block :+1:

**Review by ardelato — APPROVED** [review]
CR 👍

File: frontend/src/pull-card/signatures.tsx [hunk 1]
```diff
@@ -33,21 +33,25 @@ function Signatures({pull, title, signatures, required}: SignaturesProps) {
          title={noneToShow ? `No ${title} Required` : ''}
       >
       <Box m="2px" mr={noneToShow ? "2px" : 2}>{title}</Box>
-      {NewAndOldSigs(allSignatures)}
+      {NewAndOldSigs(allSignatures, pull)}
```
Comment (andyg0808) [inline frontend/src/pull-card/signatures.tsx:36]: At this point, might be worth making `NewAndOldSigs` into a real React component.
```suggestion
      <NewAndOldSigs allSignatures={allSignatures} pull={pull}/>
```
and add the curly braces to make `NewAndOldSigs` into an actual component.

**Comment by cdcline** [comment]
CR 👍

**Review by andyg0808 — APPROVED** [review]
One other question: I didn't dig into the API, but does GitHub not provide preconstructed links to the comments?

**Comment by andyg0808** [comment]
CR :goat: through 102f085

**Comment by deltuh-vee** [comment]
QA 🎬 
QA and CR checkboxes now link to their respective Github comments.

**Pull request merged**